ReactWindows/ReactNative.Shared/Views/TextInput/ReactPasswordBoxShadowNode.cs

```diff
@@ -179,6 +180,61 @@ public override void OnCollectExtraUpdates(UIViewOperationQueue uiViewOperationQ
         }
 
         /// <summary>
+        /// Computes paddings and sets <see cref="_computedPadding"/> variable so that
```

Comment:
Not sure how @rozele feels, but I don't see a need to add xml documentation for private methods, nevermind a short one like this. On a side note, even if this was a public method, mentioning a private field detail wouldn't be advised IMO.

Reply:
A+ for trying to follow the style in the surrounding code, though! 🥇 :)